### PR TITLE
Hard-coded SentimentRecommendations

### PR DESCRIPTION
**Pull request by ankitamitra**
All but one are random recommendations. "The Seven Husbands of Evelyn Hugo" has what we expect to be the real sentimentRecommendations. 

book-detail also includes google-able author, publication, and title

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/SentimentRecommenderServlet.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    makeHardCodedMappings();
+
+    Book book = bookDao.getEntity(request.getParameter("isbn"));
+
+    String json = gson.toJson(hardCodedMappings.get(book));
+    response.getWriter().println(json);
+  }
+
+  private void makeHardCodedMappings() {
+    hardCodedMappings = new HashMap<Book, List<Book>>();
```
Comment (cjwert-g) [inline portfolio/src/main/java/com/google/sps/servlets/SentimentRecommenderServlet.java:48]: Are these hardCodedMappings something we should build every time a get request is made, or could we initialize it when the server starts? I believe you have written logic to initialize objects when the server starts in other servlets.

Comment (ankitamitra) [reply portfolio/src/main/java/com/google/sps/servlets/SentimentRecommenderServlet.java:50]: ahh yes that makes sense!

File: portfolio/src/main/java/com/google/sps/servlets/SentimentRecommenderServlet.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  public void init() {
+    bookDao = new BookDaoDatastore();
+    makeHardCodedMappings();
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Gson gson = new Gson();
+    response.setContentType("application/json");
+    List<Book> bookList = bookDao.getBookList();
```
Comment (cjwert-g) [inline portfolio/src/main/java/com/google/sps/servlets/SentimentRecommenderServlet.java:43]: I don't see bookList used. Can we delete this line?

File: portfolio/src/main/java/com/google/sps/servlets/SentimentRecommenderServlet.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    Book book = bookDao.getEntity(request.getParameter("isbn"));
+    String json = gson.toJson(hardCodedMappings.get(book));
```
Comment (cjwert-g) [inline portfolio/src/main/java/com/google/sps/servlets/SentimentRecommenderServlet.java:45]: nit: no need for json variable. We can just combine this with the following line:
response.getWriter().println(gson.toJson(...))

**Pull request merged**